apps/competitions-api/src/routes/admins/updateAdmin.ts

```diff
@@ -0,0 +1,63 @@
+import { Router } from 'express';
+import { prisma } from '@competition-manager/prisma';
+import { parseRequest, checkRole, checkAdminRole, AuthenticatedRequest } from '@competition-manager/utils';
+import { Eid$, Id$ } from '@competition-manager/schemas';
+import { BaseAdmins$, AdminWithoutIdAndRelation$ } from '@competition-manager/schemas';
+import { z } from 'zod';
+
+export const router = Router();
+
+const Params$ = z.object({
+    competitionEid: Eid$,
+    adminId: Id$
+});
+
+const Body$ = AdminWithoutIdAndRelation$
+
+router.put(
+    '/:competitionEid/admins/:adminId',
+    parseRequest('body', Body$),
+    parseRequest('params', Params$),
+    checkRole('club'),
+    async (req: AuthenticatedRequest, res) => {
+        try{
+            const { competitionEid, adminId } = Params$.parse(req.params);
+            const newAdmin = Body$.parse(req.body);
```

Comment:
Pas utile de definir Body$
```suggestion

router.put(
    '/:competitionEid/admins/:adminId',
    parseRequest('body', AdminWithoutIdAndRelation$),
    parseRequest('params', Params$),
    checkRole('club'),
    async (req: AuthenticatedRequest, res) => {
        try{
            const { competitionEid, adminId } = Params$.parse(req.params);
            const newAdmin = AdminWithoutIdAndRelation$.parse(req.body);
```